### PR TITLE
Fix for sleeping beauty, actually sleeps now

### PR DESCRIPTION
**Pull request by fkautz**

### CODE REVIEW

File: sleeping-beauty/sleeping-beauty.asm [hunk 1]
```diff
@@ -53,8 +53,8 @@ _start:
 
 do_pause:
 	; pause()
-	mov al, 29 ; sys_pause
-	syscall
+    mov eax, 29 ; sys_pause
+    int 0x80 ; execute syscall
```
Comment (tianon) [inline sleeping-beauty/sleeping-beauty.asm]: Can you fix the indentation please? :wink:

**Comment by tianon** [comment]
Also, I'd prefer to not have the binary itself in the PR. :+1:

**Comment by fkautz** [comment]
Binary removed, the binary was present before, which is why I added it.

**Comment by tianon** [comment]
Right, that's not what I meant.  The binary _has_ to be present for this to work as an automated build.

What I meant was that I don't want any changes at all to the binary in this PR. :wink:

**Comment by fkautz** [comment]
Changed again, binary isn't modified.

**Pull request merged**